modules/installation-disk-partitioning-upi-templates.adoc

```diff
@@ -0,0 +1,117 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-user-infra.adoc
+// * installing/installing_aws/installing-restricted-networks-aws.adoc
+// * installing/installing_azure/installing-azure-user-infra.adoc
+// * installing/installing_gcp/installing-gcp-user-infra.adoc
+// * installing/installing_gcp/installing-restricted-networks-gcp.adoc
+
+// Similar content to what is in this module is also present in modules/installation-disk-partitioning.adoc. <-- This module is in use with the following vSphere assemblies:
+//    * installing-vsphere.adoc
+//    * installing-vsphere-network-customizations.adoc
+//    * installing-restricted-networks-vsphere.adoc
+
+// Similar content to what is in this module is also present in modules/installation-user-infra-machines-advanced.adoc. <-- This module is in use with the following bare metal assemblies:
+//    * installing-bare-metal-network-customizations.adoc
+//    * installing-bare-metal.adoc
+//    * installing-restricted-networks-bare-metal.adoc
+
+[id="installation-disk-partitioning-upi-templates_{context}"]
+= Creating a separate `/var` partition
+In general, disk partitioning for {product-title} should be left to the
+installer. However, there are cases where you might want to create separate partitions in a part of the filesystem that you expect to grow.
+
+{product-title} supports the addition of a single partition to attach
+storage to either the `/var` partition or a subdirectory of `/var`.
+For example:
+
+* `/var/lib/containers`: Holds container-related content that can grow
+as more images and containers are added to a system.
+* `/var/lib/etcd`: Holds data that you might want to keep separate for purposes such as performance optimization of etcd storage.
+* `/var`: Holds data that you might want to keep separate for purposes such as auditing.
+
+Storing the contents of a `/var` directory separately makes it easier to grow storage for those areas as needed and reinstall {product-title} at a later date and keep that data intact. With this method, you will not have to pull all your containers again, nor will you have to copy massive log files when you update systems.
+
+Because `/var` must be in place before a fresh installation of
+{op-system-first}, the following procedure sets up the separate `/var` partition
+by creating a machine config that is inserted during the `openshift-install`
+preparation phases of an {product-title} installation.
+
+.Procedure
+
+. Create a directory to hold the {product-title} installation files:
```

Comment:
These first two steps don't make sense in the context of the assemblies because you've already generated and modified the manifests.